app/views/manuals/updates.html.erb

```diff
@@ -1,3 +1,9 @@
+<% content_for :extra_headers do %>
```

Comment:
Perhaps this should not be the same as the manual page. Perhaps something like this?

> Overview of changes for "Content design: planning, writing and managing content"

Maybe check with a content person.